canonical-web-and-design/webteam.canonical.com
# Guides: link to discourse

## Done 

Instead of having a dedicated guides page, let's just link to discourse. The rationale for this can be found [in the spec](https://docs.google.com/document/d/1RI5N2rbasGZid1Vr2CdMhIKaEirT06PjU-bgjnENTn8/edit):

> ### The structure of the Guides
> Currently the Guides page uses a similar structure to other docs pages - it has a manually maintained side navigation, with grouping and nesting.
> As the guides are much less structured than our other docs or Practices site, it’s sometimes difficult to place a given guide in just one category, or find a category for it at all. It may be much more useful to be able to search those, or group them by keywords or tags (“docker” “k8s” “demos” “assets”).
> Various ideas to consider:
> - Adding search to the guides
> - Automatically generating list of guides from discourse API (initially proposed by [Goulin Khoge](mailto:goulin.khoge@canonical.com))
> - Showing a sortable/filterable list of guides instead of page with side navigation
> Although many of those seem to mean reimplementing some of native discourse features. So maybe it would be useful to keep Guides *just in the discourse itself*, where they can be searched for, tagged, etc.

## QA

- Open demo
- Click on Guides in top navigation
- Check it goes to the guides section on discourse 

## Issue

Fixes https://github.com/canonical/Web-design-systems-squad/issues/54
